src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -78,9 +79,9 @@ public class SkriptCommand implements CommandExecutor {
 			.add("download")
 		).add("info"
 		).add("help");
-	
+
 	static {
-		if (new File(Skript.getInstance().getDataFolder() + "/doc-templates").exists()) {
+		if (Documentation.getDocsTemplateDirectory().exists()) {
```

Comment:
```suggestion
		if (Documentation.getDocsTemplateDirectory().exists()) {
```
shouldn't this be the getter for isDocsTemplateFound? also see my comment on that method